verticapy/ml_pipeline/examples/recipe1.yaml

```diff
@@ -0,0 +1,54 @@
+schema: "public"      # Required
```

Comment:
The examples should be directly added to the doc. You should build up a Pipeline tab and add all the information.